### PR TITLE
Added missing params to Scoring Output Types

### PR DESCRIPTION
**Pull request by kacper1409**
#864

### CODE REVIEW

**Comment by kacper1409** [comment]
I checked for each of added params that after selecting it from dropdown list, there is proper output in detect.dat.

File: src/ThreeEditor/util/Scoring/ScoringOutputTypes.ts [hunk 1]
```diff
@@ -10,15 +10,25 @@ export const DETECTOR_KEYWORD_DESCRIPTION = {
...
+	'dQ': 'Dose-averaged Q',
+	'dQeff': 'as dZeff2Beta2',
 	'Energy': 'Total amount of energy deposited [MeV]',
 	'EqvDose': 'Equivalent dose (see notes below) [Sv]',
 	'Fluence': 'Fluence [/cm2]',
 	'MATERIAL': 'Maps material ID as assigned in geo.dat. Useful for debugging geometries.',
 	'NEqvDose': 'Equivalent dose from neutron kerma (see notes below) [Sv]',
 	'NKERMA': 'Neutron Kerma in [Gy]',
+	'Q': 'alias for DQ',
+	'Qeff': 'as dZeff2Beta2',
```
Comment (grzanka) [inline src/ThreeEditor/util/Scoring/ScoringOutputTypes.ts:22]:
```suggestion
```

File: src/ThreeEditor/util/Scoring/ScoringOutputTypes.ts [hunk 2]
```diff
@@ -10,15 +10,25 @@ export const DETECTOR_KEYWORD_DESCRIPTION = {
 	'DDD': 'as Dose, but specially for TRiP98 depth-dose kernel file generation',
 	'DoseGy': 'Dose [Gy]',
 	'dLET': 'Dose-averaged LET [MeV/cm]',
+	'dQ': 'Dose-averaged Q',
+	'dQeff': 'as dZeff2Beta2',
```
Comment (grzanka) [inline src/ThreeEditor/util/Scoring/ScoringOutputTypes.ts:14]:
```suggestion
	'dQeff': 'Dose-averaged Qeff',
```

File: src/ThreeEditor/util/Scoring/ScoringOutputTypes.ts [hunk 3]
```diff
@@ -10,15 +10,25 @@ export const DETECTOR_KEYWORD_DESCRIPTION = {
...
 	'MATERIAL': 'Maps material ID as assigned in geo.dat. Useful for debugging geometries.',
 	'NEqvDose': 'Equivalent dose from neutron kerma (see notes below) [Sv]',
 	'NKERMA': 'Neutron Kerma in [Gy]',
+	'Q': 'alias for DQ',
+	'Qeff': 'as dZeff2Beta2',
 	'ZONE': 'Maps zone number. Useful for debugging geometries',
 	'Rho': 'Maps material density as assigned in geo.dat and mat.dat [g/cm³]. Useful for debugging geometries.',
-	'tLET': 'Track-averaged LET [MeV/cm]'
+	'tQ': 'Track-averaged Q',
+	'tQeff': 'as tZeff2Beta2',
```
Comment (grzanka) [inline src/ThreeEditor/util/Scoring/ScoringOutputTypes.ts:26]:
```suggestion
	'tQeff': 'Track-averaged Qeff',
```

**Review by grzanka — CHANGES_REQUESTED** [review]

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**